tests/integration/Segments/DynamicSegments/Filters/WooCommerceTotalSpentTest.php

```diff
@@ -129,4 +134,15 @@ private function cleanUp(): void {
       $this->wp->wpDeletePost($orderId);
     }
   }
+
+  private function setIsWoocommerceUser(array $emails): void {
```

Comment:
I'm inclined to think that we should move this method somewhere else so that we don't have to repeat it here and in WooCommerceNumberOfOrdersTest. Have you considered this option?

Reply:
I was thinking about a method in the repository, but I don't see any usage. In my opinion, this case is rare, so I created this method twice.